docs/development/release_management.rst

```diff
@@ -286,9 +286,9 @@ Release Process
 #. A reviewer must verify the build logs, obtain and sign the generated ``Release``
    file, and append the detached signature to the PR. The PR should remain in
    draft mode. The packages on ``apt-qa.freedom.press`` are now installable.
-#. Coordinate with one or more team members to confirm a successful
-   clean install in production VMs using the packages on
-   ``apt-qa.freedom.press``.
+#. Coordinate with one or more team members to `confirm a successful clean install in production VMs
+   <https://github.com/freedomofpress/securedrop/wiki/QA-Procedures#user-content-preflight-testing>`__
```

Comment:
Turns out this change to the anchor ID is necessary because GitHub only pretends that its anchors are anchors -- they're actually resolved via JavaScript, and the underlying anchor ID is hidden from view. (As a side effect, trying to visit an anchored GitHub URL without JavaScript means -- no working anchor. Nice one, GitHub!)